lib/msal-core/src/AuthError.ts

```diff
@@ -0,0 +1,240 @@
+/**
+  * Copyright (c) Microsoft Corporation
```

Comment:
1 class per file please 
So much easier to read 
lets add a src/error folder 
index.ts // Exports all the classes so they can be imported like this
```js
import { AuthError, InteractionError } from './errors';
```

then have files fore
AuthError.ts
ClientInteractionError.ts
ServerError.ts
 etc...

Reply:
Ok I will update and separate these files.